api/lanes.py

```diff
@@ -608,26 +529,19 @@ class ContributorLane(QueryGeneratedLane):
     ROUTE = 'contributor'
     MAX_CACHE_AGE = 48*60*60    # 48 hours
 
-    def __init__(self, _db, contributor_name, contributor_id=None,
```

Comment:
I agree, and I'd prefer to improve our Contributor data before we start treating identifiers as actually unique people. I'd like to leave it like this for the time being, but we can consider fixing it if we see problems.